src/components/UserMenu/UserMenu.types.ts

```diff
@@ -19,26 +19,25 @@ export type UserMenuProps = Omit<
 
 export type UserMenuI18N = Record<
   | 'myAssets'
-  | 'settings'
   | 'account'
   | 'viewProfile'
   | 'signIn'
   | 'signOut'
   | 'guest'
-  | 'wallet'
+  | 'marketplaceAuthorizations'
   | 'jumpIn',
   string
 >
 
 export enum UserMenuEventId {
   ACTIVITY = 'activity',
+  ACCOUNT = 'account',
   MY_ASSETS = 'my_assets',
-  SETTINGS = 'settings',
   PROFILE = 'profile',
   SIGN_IN = 'sign_in',
   SIGN_OUT = 'sign_out',
   GUEST = 'guest',
-  WALLET = 'wallet',
+  MARKETPLACE_AUTHORIZATION = 'marketplace_authorization',
```

Comment:
Shouldn't this be in its plural form?
```suggestion
  MARKETPLACE_AUTHORIZATIONS = 'marketplace_authorizations',
```